### PR TITLE
Check to make sure some backup files actually exist before running tar

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: templates/default/upload-incrementals.erb [hunk 1]
```diff
@@ -32,7 +32,8 @@ for data_dir in $data_dirs; do
 
     tarball="${work_dir}/$keyspace-$dir_index.tar.gz"
 
-    if [ -n "$(find $data_dir/$keyspace -maxdepth 2 -mindepth 2 -name backups -type d | head -n 1)" ]; then
+    if [ -n "$(find $data_dir/$keyspace -maxdepth 2 -mindepth 2 -name backups -type d | head -n 1)" ] && \
+       [ "$(ls -A $data_dir/$keyspace/*/backups/*)" ]; then
```
Comment (jeffbyrnes) [inline templates/default/upload-incrementals.erb]: @eherot so this should:

1. ensure `$data_dir/$keyspace` has at least 1 dir
2. ensure `$data_dir/$keyspace/*/backups/*`has real content (i.e., excluding `.` and `..`)

Comment (eherot) [inline templates/default/upload-incrementals.erb]: 1. At least one dir *named backup*
2. Yes

**Comment by jeffbyrnes** [comment]
:+1: from me, merge & ship when ready! Though bear in mind that `et_contacts_cassandra` depends on `et_cassandra ~> 4.0`

**Comment by eherot** [comment]
Actually `~> 4.7`. Is that a problem?

**Comment by jeffbyrnes** [comment]
@eherot it shouldn’t be, though it will allow anything other than major releases of this cookbook to go straight to prod. Feel free to tighten it if you like.

**Pull request merged**